src/HealthChecks.Gcp.CloudStorage/HealthChecks.Gcp.CloudStorage.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Copyright></Copyright>
+    <Description>HealthChecks.Gcp.CloudStorage is the health check package for the CloudStorage Buckets inside ProjectIds.</Description>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <RepositoryUrl>http://github.com/xabaril/AspNetCore.Diagnostics.HealthChecks</RepositoryUrl>
+    <PackageIconUrl />
+    <RepositoryType></RepositoryType>
+    <PackageId>AspNetCore.HealthChecks.Gcp.CloudStorage</PackageId>
+    <Version>3.0.0</Version>
+    <Authors>Xabaril Contributors</Authors>
+    <Company>Xabaril</Company>
+    <Product>HealthChecks.Gcp.CloudStorage</Product>
+    <PackageReleaseNotes></PackageReleaseNotes>
+    <AssemblyVersion>3.0.0.0</AssemblyVersion>
```

Comment:
Consider using single `<VersionPrefix>3.0.0</VersionPrefix>` instead of multiple versions.
Also consider moving all copy-pasted settings from csproj files into single `Directory.Build.props` root file.

Reply:
I did not quite copy pasted , it's actually Visual Studio 2019 default if you fill both assembly version and version. At least I think 🤔

Reply:
My comment does not apply directly to your PR. Just with every new health check project in this repo I see the same copied configuration file. We can finally put it in one place.

Reply:
I think everyone copies it because to be sure complies with the repo need to be declared as healthchecks contributors to be sure it's ok plus the version 😁. But ok , that file should be in the main master repo I think being there will make at build all projects with same versions and comments etc...

Reply:
I am guided by the DRY principle.